src/jit/compiler.h

```diff
@@ -7466,6 +7466,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #pragma endregion // Note: region is NOT under !defined(__GNUC__)
 #endif
 
+    bool IsPreJIT()
+    {
+        return jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT);
```

Comment:
There are a lot of places in the JIT that just do `jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT);` directly, it might be useful to update those to call `IsPreJIT()` post the 3.0 snap.